README.md

```diff
@@ -47,7 +47,8 @@ The main configuration options are listed below:
 | SEFT_FTP_PORT                         | `2021`                         | FTP port
 | SEFT_FTP_USER                         | `ons`                          | FTP username
 | SEFT_FTP_PASS                         | `ons`                          | FTP password
-| SEFT_CONSUMER_FTP_FOLDER              | `.`                            | FTP Folder
+| SEFT_CONSUMER_FTP_FOLDER              | `.`                            | FTP Folder (Deprecated)
```

Comment:
Lets just remove it rather than deprecated it

We'll release v2.0.0 when this is merged instead

Reply:
Changed